test/public/envs/overview.test.js

```diff
@@ -177,4 +178,33 @@ module.exports = () => {
         await waitForNavigation(page, () => pressElement(page, '#rowTDI59So3d-runs a'));
         expectUrlParams(page, { page: 'run-detail', runNumber: 103 });
     });
+
+    it('should successfully display dropdown links', async () => {
+        await goToPage(page, 'env-overview');
```

Comment:
No more goToPage except for new test suite.

Reply:
applied